python/paddle/fluid/dygraph/dygraph_to_static/call_transformer.py

```diff
@@ -36,8 +36,9 @@ def _is_builtin_call(self, node):
         assert isinstance(node, gast.Call)
         func_str = ast_to_source_code(node.func).strip()
         try:
-            from paddle.fluid.dygraph.dygraph_to_static.convert_call_func import is_builtin
-            return eval("is_builtin({})".format(func_str))
+            from paddle.fluid.dygraph.dygraph_to_static.convert_call_func import is_builtin_len, is_builtin
+            return not eval("is_builtin_len({})".format(func_str)) and eval(
```

Comment:
The logic is still different to the method name `_is_builtin_call` at line 35, could you change it?

Reply:
Thanks, done~~